<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KDE SPLASH CREATOR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Elements:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_remove">
        <property name="text">
         <string>REMOVE ELEMENT</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_elements">
      <property name="currentText">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_add">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>ADD ELEMENT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_background">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>BACKGROUND OPTIONS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_splashinfo">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>SET SPLASH INFO</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_save">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>SAVE TO</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_apply">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>APPLY SPLASH </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
